### ISSUE TITLE
Dashlet search improvements needed

### ISSUE THREAD

**Issue by pgorod** [salesagility/SuiteCRM#1313]
When adding new Dashlets to the Home screen, there is a box to search dashlets. After being surprised by not being able to find "My activity stream" and "JotPad" dashlets when searching, I realized it's searching each tab separately. So I would need to know that these are "Tools", not "Modules", and search on that tab. But the point of having a search is to not make people look for it, or think how somebody classified it... : - )

The code is in [MySugar.php](https://github.com/salesagility/SuiteCRM/blob/5d002c964fc984bad892cb73456849f051762caa/include/MySugar/MySugar.php).

I suggest an enhancement, searching all tabs at once. While this is a minor issue, it is one that end-users actually use by themselves, so it's enough to get people bothering Support because they can't find the dashlet they just accidentally clicked to close.

Also, the search box shouldn't be cleared when moving from tab to tab, as it seems to be now (I actually have to type my search twice to search in more than one tab).

For me the perfect behavior for this search box would be to make it find-as-you-type across all tabs, but that might be too much to ask.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Dashlet search improvements needed](https://trello.com/c/d9kaH7CL/100-dashlet-search-improvements-needed)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
